deploy/clowdapp.yaml

```diff
@@ -2904,6 +2906,9 @@ parameters:
 - displayName: RBAC_CACHE_TIMEOUT
   name: RBAC_CACHE_TIMEOUT
   value: "300"
+- displayName: Middleware Timeout
+  name: CACHE_TIMEOUT
+  value: "300"
```

Comment:
Let's do 3600 to match the settings file to make this a no-change change in cases where we don't explicitly set this var in app interface.

Reply:
Doh, I meant to do that. 🤦